sdcm/nemesis.py

```diff
@@ -3357,8 +3359,12 @@ def is_virtual_tables_get_snapshot():
                 with self.cluster.cql_connection_patient(self.cluster.nodes[0]) as session:
                     for cf in snapshot_params[3].split(','):
                         keyspace_table.extend([[keyspace, cf]])
-                        for view in get_views_of_base_table(keyspace_name=keyspace, base_table_name=cf, session=session):
-                            keyspace_table.extend([[keyspace, view]])
+                        # Stop taking snapshots of MVs when snapshotting a base table - presented in
+                        # https://github.com/scylladb/scylladb/pull/21433/commits/9645a0414dbf4f41e0ce612b485b09c16a408f42
+                        if not (ComparableScyllaVersion(self.target_node.scylla_version) >= "2025.1.0" or
```

Comment:
`2025.1.0` is newer than the change landed to.
For example, current `2025.1.0-rc2` won't pass this condition.

Also, the `6.3.0-dev` is enough according to the fact of latest license changes for ScyllaDB product.

Reply:
changed